### PR TITLE
New Codio Cheat Sheet

### PR DESCRIPTION
**Pull request by riqpe**
https://duck.co/ia/view/codio_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@riqpe Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

While the team and the community review your submission, we'd like to make sure you look good! If you'd like a free DuckDuckGo tshirt, please send over the following info to merch@duckduckgo.com, and be sure to put "Dev Merch" in the subject line:

Name:
Street Address:
City:
State / Province:
Country:
Phone (Non-US Orders only): 
T-shirt size:
GitHub username:
Link to this Pull Request:

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/codio.json [hunk 1]
```diff
@@ -0,0 +1,278 @@
+{
+	"id": "codio_cheat_sheet",
+	"name": "Codio Shortcuts",
+	"description": "The default set of keyboard shortcuts to use Codio efficiently. You can customise these by editing your User preferences, which can be found in Codio > Preferences -> User.",
+
+	"aliases": [
+		"Codio cheat sheet", "Codio shortcuts"
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/codio.json]: Get rid of these aliases - they are covered by triggers.

File: share/goodie/cheat_sheets/json/codio.json [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+			"key": "Ctrl+Space",
+			"val": "Show auto-completions."
+		}, {
+			"key": "Ctrl+/",
+			"val": "Toggle comment."
+		}, {
+			"key": "Alt+Left",
+			"val": "Go to start of line."
+		}, {
+			"key": "Alt+Right",
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/codio.json]: Lose the '+'s, multiple keys together are implicit.

File: share/goodie/cheat_sheets/json/codio.json [hunk 3]
```diff
@@ -0,0 +1,278 @@
...
+		"Search": [{
+			"key": "Ctrl+H",
+			"val": "Find in project"
+		}, {
+			"key": "Ctrl+Shift+H",
+			"val": "Replace in project"
+		}],
+		"IDE": [{
+			"key": "Alt+Shift+N",
+			"val": "New file."
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/codio.json]: Lets lose the periods after the vals, don't really need them.

**Comment by riqpe** [comment]
Thanks for the feedback. I'll fix those issues later tonight.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Codio cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/codio_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Codio cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/codio_cheat_sheet_mobile.png)

File: share/goodie/cheat_sheets/json/codio.json [hunk 4]
```diff
@@ -0,0 +1,274 @@
...
+		"Deployment",
+		"IDE",
+		"Code-beautifier",
+		"Guides",
+		"Search"
+	],
+
+	"sections": {
+		"Editor": [{
+			"key": "Ctrl Space",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/codio.json]: @riqpe You need to wrap individual keys in `[]`. i.e. `[Ctrl] [Space]`

Please ensure all individual keys are wrapped. Take a look at other Cheat Sheets to see how they're written :+1:

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Codio cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/codio_cheat_sheet_mobile.png)

**Comment by riqpe** [comment]
I see what you mean. Mea culpa. Should be fixed now.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Codio cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/codio_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Codio cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/codio_cheat_sheet_mobile.png)

File: share/goodie/cheat_sheets/json/codio.json [hunk 5]
```diff
@@ -0,0 +1,274 @@
+{
+	"id": "codio_cheat_sheet",
+	"name": "Codio Shortcuts",
+	"description": "The default set of keyboard shortcuts to use Codio efficiently. You can customise these by editing your User preferences, which can be found in Codio > Preferences -> User.",
+
+	"template_type": "keyboard",
+
+	"section_order": [
+		"Editor",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/codio.json]: nit: please fix the indentation -- we prefer 4-space soft tabs (spaces)

**Comment by moollaza** [comment]
@riqpe Thanks! This LGTM :+1:

**Pull request merged**